traffic_portal/app/src/common/modules/form/physLocation/FormPhysLocationController.js

```diff
@@ -42,7 +42,7 @@ var FormPhysLocationController = function(physLocation, $scope, $location, formU
         { name: 'shortName', type: 'text', required: true, maxLength: 100 },
         { name: 'address', type: 'text', required: true, maxLength: 128 },
         { name: 'city', type: 'text', required: true, maxLength: 128 },
-        { name: 'state', type: 'text', required: true, maxLength: 2 },
+        { name: 'state', type: 'text', required: true, maxLength: 4 },
```

Comment:
This should match the limitations set by the API - and currently there is no max length in the API. You might arbitrarily choose something huge like 255 but 5+ characters is actually totally valid and our front-end ought to model that more accurately.

Reply:
Should we also set the limit in Traffic Ops API?

Reply:
You could do that, but that would be a breaking API change which would necessitate an APIv6, deprecation strategy, and a plan on how to eventually migrate old data to the new, more restrictive format.

It'd be a lot less work to just eliminate the requirement altogether in the UI.